tests/Operation/BulkWriteFunctionalTest.php

```diff
@@ -161,82 +163,80 @@ public function testUnacknowledgedWriteConcern()
 
     /**
      * @depends testUnacknowledgedWriteConcern
-     * @expectedException MongoDB\Exception\BadMethodCallException
-     * @expectedExceptionMessageRegExp /[\w:\\]+ should not be called for an unacknowledged write result/
      */
     public function testUnacknowledgedWriteConcernAccessesDeletedCount(BulkWriteResult $result)
     {
+        $this->expectException(BadMethodCallException::class);
+        $this->expectExceptionMessageRegExp('/[\w] should not be called for an unacknowledged write result/');
```

Comment:
I realize that `'/[\w:\\]+ should not...` is what was originally in the `@expectedExceptionMessageRegExp`, but when I use that regular expression in `$this->expectExceptionMessageRegExp()`, I get a `PHPUnit\Framework\Exception: Invalid expected exception message regex given`.

Reply:
Ah! I think that's because `\\` in the PHP string translates to a single, escaped backslash. Regex then thinks we're escaping the follow `]`. Since we need to provide `\\` to the regex parser, we may need to use `\\\\` in the PHP string.